pkg/api/v1alpha1/edgeconnect/edgeconnect_types.go

```diff
@@ -116,6 +123,11 @@ type ProxySpec struct {
 	Port uint32 `json:"port,omitempty"`
 }
 
+type KubernetesApiAutomationSpec struct {
+	// Enabled enables kubernetes automation
```

Comment:
```suggestion
	// Enables kubernetes automation
```

do we have to say the name of the field 😅

Reply:
fixed